sdk/eventhub/Azure.Messaging.EventHubs.Processor/src/EventProcessorClient.cs

```diff
@@ -547,6 +547,22 @@ protected EventProcessorClient()
         /// <exception cref="InvalidOperationException">Occurs when this method is invoked without <see cref="ProcessEventAsync" /> or <see cref="ProcessErrorAsync" /> set.</exception>
         ///
         public virtual async Task StartProcessingAsync(CancellationToken cancellationToken = default)
+            => await StartProcessingImplAsync(true, cancellationToken).ConfigureAwait(false);
```

Comment:
If you'd be so kind, please move the new `private` methods to the private section.   These are organized by visibility, starting with public and then in decreasing order.

Reply:
Fixed